src/applications/disability-benefits/all-claims/content/form0781/workflowChoicePage.jsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+
+export const workflowChoicePageTitle = (
+  <div className="vads-u-font-size--h3">
+    Statement about mental health conditions (VA Form 21-0781)
+  </div>
+);
+
+// Lists new conditions the veteran has claimed
+const conditionSelections = formData => {
+  const conditions = formData?.newDisabilities?.map(
+    disability =>
+      // Capitalize condition
+      disability.condition.charAt(0).toUpperCase() +
+      disability.condition.slice(1),
+  );
+
+  if (conditions?.length) {
+    return (
+      <p>
+        {/* Leave space ahead of conditions list */}
+        You selected these new conditions for your disability claim:{' '}
+        <strong>{conditions.join(', ')}</strong>.
```

Comment:
I know that design has comma separates conditions but, after viewing the screenshot provided in the PR description, I wonder if that choice should be reconsidered as there are conditions that also include commas and, when joined with other conditions, the differentiation may be confusing. Just mentioning for consideration.

Reply:
Yeah I noticed that as well meant to mention that in the PR, it's less confusing because there the conditions are capitalized but it's still confusing. I'm not how we get around that, unless we revise this and list the conditions in bullet points, I can ask Ruben about that

Reply:
Yes we will do bulleted list here

Reply:
Made the change for this [here](https://github.com/department-of-veterans-affairs/vets-website/pull/33655/commits/320f881724502c90482556dc04628e209e440c6e) and updated the screenshot in the PR description